tests/CMakeLists.txt

```diff
@@ -1,15 +1,41 @@
-function(add_cmake_test_test)
-    foreach(testi ${ARGN})
-        set(full_path ${CMAKE_CURRENT_LIST_DIR}/${testi}.cmake)
-        add_test(
-            NAME "${testi}"
-            COMMAND ${CMAKE_COMMAND}
-            -DCMAKE_MODULE_PATH=${PROJECT_SOURCE_DIR}
-            -P ${full_path}
-        )
-    endforeach()
-endfunction()
-
-add_subdirectory(asserts)
-add_subdirectory(detail_)
-add_subdirectory(tutorials)
+cmake_minimum_required(VERSION "${_ct_min_cmake_version}") #Required for FetchContent_MakeAvailable()
+
+project(CMakeTest LANGUAGES C) #Need C language so CMake doesn't complain about no linker language for some empty files used for tests
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../")
+set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}" CACHE STRING "" FORCE)
+
+
+#Enable colors in Unix environments, ignored on Windows. Will output garbage for pipes and text files
+set(CMAKETEST_USE_COLORS "FALSE")
```

Comment:
I think you want [`option`](https://cmake.org/cmake/help/latest/command/option.html) here otherwise you'll override the user's selection.

Reply:
That is exactly what I needed yes, I'll change that soon

Reply:
Should be fixed now